src/features/game/expansion/components/IslandList.tsx

```diff
@@ -69,9 +73,36 @@ const Island = ({ name, levelRequired, path, bumpkin, image }: Props) => {
   );
 };
 
-export const IslandList = ({ bumpkin }: { bumpkin: Bumpkin | undefined }) => {
+const VisitFriendListItem: React.FC<{ onClick: () => void }> = ({
+  onClick,
+}) => {
+  return (
+    <div onClick={onClick}>
+      <OuterPanel className="flex relative items-center py-2 mb-1 cursor-pointer hover:bg-brown-200">
+        <div className="w-16 justify-center flex mr-2">
+          <img src={land} className="h-9" />
+        </div>
+        <div className="flex-1 flex flex-col justify-center">
+          <span className="text-sm">Visit Friend</span>
+        </div>
+      </OuterPanel>
+    </div>
+  );
+};
+
+export const IslandList = ({
+  bumpkin,
+  showVisitList,
+}: {
+  bumpkin: Bumpkin | undefined;
+  showVisitList: boolean;
+}) => {
+  const { authService } = useContext(Auth.Context);
+  const [authState, send] = useActor(authService);
   const { id } = useParams();
   const location = useLocation();
+  const [view, setView] = useState<"list" | "form">("list");
```

Comment:
What is a form for the IslandList? If this is visiting related can we make this a bit more specific.

E.g. `visitForm`